ctutil/fuzz_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2024 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ctutil
+
+import (
+	"crypto/x509"
+	ctx509 "github.com/google/certificate-transparency-go/x509"
+	"github.com/google/certificate-transparency-go/x509util"
+	"testing"
+)
+
+func FuzzVerifySCTTest(f *testing.F) {
+	f.Fuzz(func(t *testing.T, publicKey, certData []byte) {
+		cert, err := x509.ParseCertificate(certData)
+		if err != nil {
+			t.Skip()
+		}
+		scts, err := x509util.ParseSCTsFromCertificate(cert.Raw)
+		if err != nil {
+			t.Skip()
+		}
+		chain, err := ctx509.ParseCertificates(cert.Raw)
+		if err != nil {
+			t.Skip()
+		}
+		pub, err := x509.ParsePKIXPublicKey(publicKey)
+		if err != nil {
+			t.Skip()
+		}
+		panic("HERE")
```

Comment:
Is this intentional?

Reply:
Thanks for catching that. No, it wasn't.